### DISCUSSION TITLE
markdown bug with checkbox and `

### DISCUSSION THREAD

**Discussion by matart15** [requarks/wiki#3540] [Error / Bug Report]
**Describe the bug**
I am using version `2.4.107`

markdown with checkbox and ` renders twice

**To Reproduce**
1. create page 
2. select markdown editor
3. input 
```
 - [ ] Modal内に `name`, description inbox と owner selectbox とSubmitボタンがある
```

**Expected behavior**
preview must display

- [ ] Modal内に `name` , description inbox と owner selectbox とSubmitボタンがある

but it renders `Modal内に name` twice 

**Screenshots**
![Screen Shot 2020-08-20 at 10 19 42](https://user-images.githubusercontent.com/17027219/90706108-65aa7900-e2cf-11ea-86f0-6b605da32de9.png)
![Screen Shot 2020-08-20 at 10 19 47](https://user-images.githubusercontent.com/17027219/90706114-69d69680-e2cf-11ea-91e7-7b02ad329f3e.png)

** update ** 
At first I thought it is problem with japanese language. But it happens on all english input too.

**Comment by leandro-hermes** [comment]
This problem happens when the task list item have some marked text. I had with simple bold text
![image](https://user-images.githubusercontent.com/13833942/110363377-4253db00-8021-11eb-9acb-15f312e034db.png)

**Comment by jodelm** [comment]
Using checkbox with kbd tag is OK in preview :
![image](https://user-images.githubusercontent.com/6968961/167623394-208b3719-862b-41ac-90e8-1a189519d295.png)


but not in view :
![image](https://user-images.githubusercontent.com/6968961/167624166-19227c0f-46e1-4db8-865b-b430a5d8b821.png)
